website/guides/07_kube-prom-stack.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Install the Kube Prometheus Stack
+slug: kube-prometheus-stack
+description: 'Learn how to easily install the Kube-Prometheus-Stack with Glasskube'
+authors: [ jpage ]
+tags: [ kube-prometheus-stack ]
+---
+
+import CustomGitHubButton from '../src/components/buttons/CustomGitHubButton.tsx';
+import Discussion from '@site/src/components/GiscusWrapper';
+import Install from '../src/partials/_install.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+[Prometheus](https://prometheus.io/) and [Grafana](https://grafana.com/) are open-source, community-backed solutions with stellar reputations. They bring immense value by fetching and storing metrics while enabling the creation of dashboards that are not only useful but also easy on the eyes.
+
+The uncomfortable truth is that anyone who has ever set up `Prometheus` alongside `Grafana` as their environment's monitoring stack from scratch has probably felt the frustration of not getting value especially quickly. Metric exporter configuration, dashboard widget customisation and deciding what to monitor and alert on in the first place takes time.
+
+That's why [Kube-Prometheus-Stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) was created. It installs a collection of Kubernetes manifests, Grafana dashboards, and Prometheus rules, providing an easy-to-operate, end-to-end Kubernetes cluster monitoring solution with Prometheus using the Prometheus Operator.
+
+This sounds like good news, and it is, but the stack is bundled in a Helm chart, and just the `values.yaml` file has _over 4000 lines_. Configuring and maintaining the Helm chart isn’t necessarily straightforward or “fun.”
+
+With so many configuration options, we must be getting something good right? Well yeah, we are, by deploying kube-prometheus-stack we get all of this right out of the box: 
+
+![stack-diagram](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/2fcquzgd73xf9cc83ap1.png)
+
+**Top Layer:**
+
+- **User:** Interacts with Grafana and Kubernetes API.
+
+**Visualization and Alerting Layer:**
+
+- **[Grafana](https://grafana.com/):** Connects to Prometheus for data visualization.
+- **[Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/):** Connected to Prometheus for alert management.
+- **[Prometheus Server](https://github.com/prometheus/prometheus):** Central component collecting and storing metrics.
+- **[ServiceMonitors & PodMonitors](https://observability.thomasriley.co.uk/prometheus/configuring-prometheus/using-service-monitors/):** Define how Prometheus scrapes metrics.
```

Comment:
The last two are not shown in the diagram above, is this on purpose?

Reply:
It was later going through that docs that I saw that these two components are also included, this I'll remove them just to avoid confusion.